.env.test

```diff
@@ -5,9 +5,8 @@ FLASK_DEBUG=1
 FLASK_SKIP_DOTENV=True
 
 # Change the following values for local development.
-trello_key=kghaksbnsjlaksjdlaksjdldkjlmbsanmbd
-trello_token=asdkjalskjdlaskjdaskjdaskljdlaskjdaskjdaksjdlakjdlaskjdlaskjdlaskjd
-trello_boardid=abcdefg
-todo_list_id=abcdefghijklmnopqrstuvwxyz
-doing_list_id=abcdefghijklmnopqrstuvwxyz
-done_list_id =abcdefghijklmnopqrstuvwxyz
\ No newline at end of file
+MONGO_CONN=mongodb+srv://rajrahman:NWyZIpz9kFnOW4dn@corndel.gyf3r.mongodb.net/test?w=majority
```

Comment:
This looks potentially real - is this your actual username/pass?
If so, make sure to remove this and change your DB password. Either way, I'd suggest making this string obviously fake to ensure nobody in future needs to worry about whether or not it's real